headers/types/files/wan.h

```diff
@@ -1,14 +1,99 @@
 #ifndef HEADERS_TYPES_FILES_WAN_H_
 #define HEADERS_TYPES_FILES_WAN_H_
 
-// For now, there is only the top-level header for WAN files.
+#include "../common/generic.h"
 
-// Directly loaded from memory as they are stored. Names comes from the pmd_wan project.
-#pragma pack(push, 1)
+enum wan_sprite_type {
+    PropsUI = 0, // for object and UI objects,
+    Chara = 1,   // for monster sprites,
+    Unk2 = 2,
+    Unk3 = 3
+};
+
+#pragma pack(push, 2)
+ENUM_16_BIT(wan_sprite_type);
+#pragma pack(pop)
+
+struct rgba8 {
```

Comment:
It seems like we could just reuse `struct rgb` from `file_io.h` and rename the `unused` field to `alpha`, instead of creating a new struct.

Reply:
I’ll rename this struct to RGBA and comment that alpha is sometimes unused then.